chia/pools/pool_wallet.py

```diff
@@ -778,29 +794,54 @@ async def claim_pool_rewards(self, fee: uint64) -> TransactionRecord:
             self.log.info(
                 f"Farmer coin: {coin_record.coin} {coin_record.coin.name()} {coin_to_height_farmed[coin_record.coin]}"
             )
-        if len(all_spends) == 0:
+        if len(all_spends) == 0 or the_coin_record_in_question is None:
```

Comment:
Would you be ok with renaming the_coin_record_in_question to current_coin_record? it just seems cleaner

Reply:
Good idea. Done.